### PR TITLE
fix test log time format and tag prefix.

### PR DESCRIPTION
**Pull request by hicqu**
Before:  18/05/21 May:05:1526892259.946 peer.rs:1858... ()
After   : 2818/05/21 16:45:51.855 snap.rs:747...

### CODE REVIEW

File: tests/util/mod.rs [hunk 1]
```diff
@@ -72,13 +72,13 @@ impl Drain for CaseTraceLogger {
     type Ok = ();
     type Err = slog::Never;
     fn log(&self, record: &Record, values: &OwnedKVList) -> Result<Self::Ok, Self::Err> {
-        let tag = util::get_tag_from_thread_name().unwrap_or_else(|| "".into());
+        let tag = util::get_tag_from_thread_name().map_or_else(|| "".into(), |s| s + " ");
 
         let t = time::now();
-        let time_str = time::strftime("%y/%m/%d %h:%m:%s.%f", &t).unwrap();
```
Comment (breeswish) [inline tests/util/mod.rs]: should be %Y?

Comment (hicqu) [reply tests/util/mod.rs]: `%Y` will print `2018`. `%y` will print `18`. I think both are OK.

**Comment by breeswish** [comment]
LGTM

File: tests/util/mod.rs (hunk 1 above)
Comment (BusyJay) [reply tests/util/mod.rs]: `%Y` is better, because we have been using the format elsewhere.

File: tests/util/mod.rs [hunk 2]
```diff
@@ -71,21 +71,20 @@ struct CaseTraceLogger {
 impl Drain for CaseTraceLogger {
     type Ok = ();
     type Err = slog::Never;
-    fn log(&self, record: &Record, values: &OwnedKVList) -> Result<Self::Ok, Self::Err> {
-        let tag = util::get_tag_from_thread_name().unwrap_or_else(|| "".into());
+    fn log(&self, record: &Record, _: &OwnedKVList) -> Result<Self::Ok, Self::Err> {
```
Comment (AndreMouche) [inline tests/util/mod.rs]: Should we consider the values(key-value lists of a `Logger` it was executed on)?

Comment (hicqu) [reply tests/util/mod.rs]: Seems we didn't use this feature. For now `values` always be `()`.

**Comment by siddontang** [comment]
```
test raftstore::store::util::tests::test_lease ... thread 'raftstore::store::util::tests::test_lease' panicked at 'assertion failed: `(left == right)`
  left: `Valid`,
 right: `Expired`', src/raftstore/store/util.rs:450:9
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
             at libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1: std::sys_common::backtrace::print
             at libstd/sys_common/backtrace.rs:71
             at libstd/sys_common/backtrace.rs:59
   2: std::panicking::default_hook::{{closure}}
             at libstd/panicking.rs:207
   3: std::panicking::default_hook
             at libstd/panicking.rs:223
   4: tikv::util::panic_hook::track_hook::{{closure}}
             at src/util/panic_hook.rs:59
   5: <std::thread::local::LocalKey<T>>::try_with
             at /checkout/src/libstd/thread/local.rs:290
   6: <std::thread::local::LocalKey<T>>::with
             at /checkout/src/libstd/thread/local.rs:244
   7: tikv::util::panic_hook::track_hook
             at src/util/panic_hook.rs:53
   8: core::ops::function::Fn::call
             at /checkout/src/libcore/ops/function.rs:73
   9: std::panicking::rust_panic_with_hook
             at libstd/panicking.rs:403
  10: std::panicking::begin_panic_fmt
             at libstd/panicking.rs:349
  11: tikv::raftstore::store::util::tests::test_lease
             at src/raftstore/store/util.rs:450
  12: tikv::__test::TESTS::{{closure}}
             at src/raftstore/store/util.rs:429
  13: core::ops::function::FnOnce::call_once
             at /checkout/src/libcore/ops/function.rs:223
  14: <F as alloc::boxed::FnBox<A>>::call_box
             at libtest/lib.rs:1453
             at /checkout/src/libcore/ops/function.rs:223
             at /checkout/src/liballoc/boxed.rs:635
  15: __rust_maybe_catch_panic
             at libpanic_unwind/lib.rs:102
FAILED
```

@overvenus

**Pull request merged**